frontend/locales/de.json

```diff
@@ -49,7 +49,40 @@
     "team-members": "Teammitglieder",
     "room-settings": "Raumeinstellungen",
     "add-team-member": "Teammitglied hinzufügen",
-    "add-members-ready": "Bereit"
+    "add-members-ready": "Bereit",
+    "role_taken": {
+      "title": "Rolle war bereits vergeben!",
+      "description": "{member} wurde wieder Developer."
+    },
+    "infoBox": {
+      "title": "Scrum Rollen",
+      "content": "In einem Scrum-Team gibt es drei Hauptrollen:",
+      "roles": [
+        "Product Owner",
+        "Scrum Master",
+        "Entwicklungsteam (Developer)"
+      ],
```

Comment:
didn't know that this syntax is possible. and i can't seem to find this in the docs: how are these values accessed?